metadata/society/law_enforcement

```diff
@@ -0,0 +1,76 @@
+# Title
+
+Utah Law Enforcement
+
+<!--- thought: should the title say "Utah Law Enforcement Facilities"? That's what it says in the link, but on the data page it is just "Utah Law Enforcement" which seems vague to me. -->
+
+## ID
+
+01008931-c755-4df7-aea0-b71aded68f5a
+
+## Brief Summary
+
+Point locations of facilities where sworn officers of a law enforcement agency are regularly stationed.
+
+## Summary
+
+Utah Law Enforcement contains locations where law enforcement officers are regularly based or stationed. Law enforcement agencies may include municipal police, county sheriffs, school police, and federal law enforcement agencies. Additionally, this dataset includes departments within non-law enforcement federal agencies charged with law enforcement (e.g., US Postal Inspectors), and cross jurisdictional authorities (e.g., Port Authority Police).
+
+## Description
+
+### What is the dataset?
+
+Law enforcement agencies "are publicly funded and employ at least one full-time or part-time sworn officer with general arrest powers". This is the definition used by the US Department of Justice - Bureau of Justice Statistics (DOJ-BJS) for their Law Enforcement Management and Administrative Statistics [(LEMAS)](https://bjs.ojp.gov/data-collection/law-enforcement-management-and-administrative-statistics-lemas) survey. Although LEMAS only includes non-federal agencies, this dataset includes locations for federal, state, local, and special jurisdiction law enforcement agencies.
+
+<!--- LEMAS was mentioned in the original dataset, so I kept it in, but other than being another helpful resource for those looking for law enforcement data, I'm not sure why it was specifically mentioned. -->
+
+### What is the purpose of the dataset?
+
+This dataset was created to provide a general purpose database of law enforcement facility locations. Law enforcement facility data can been used to asses whether the total police capability of a given area is adequate and to help local governments identify areas of greatest concern. The data are also a helpful resource for situational awareness planning, emergency response, and route analysis.
+
+### What does the dataset represent?
+
+This dataset is comprised of point data indicating the locations of law enforcement facilities throughout the state of Utah. Each point includes attributes such as the facility name, street address, and date it was added to the dataset (indicated by the CONTDATE field). Points have been placed on or near the main structure of the facility.
+
+### How was the dataset created?
+
+<!--- Gathering more information on this. -->
```

Comment:
@gregbunce Do you have more info on this?

Reply:
The law enforcement boundaries are created programmatically using a [python script](https://github.com/gregbunce/NextGen911UtahGISDataLoader/blob/798e52335e69af2f1a761aa245fc6ecfe619f9b6/NextGen911DataLoader/commands/LoadLawEnforcement.cs). The boundaries are composed of SGID County boundaries and SGID Municipal boundaries. The sheriff offices are built from County boundaries and the police depts are built from the Muni boundaries.

Reply:
Thanks for the info! Just want to make sure--does this all refer to the law enforcement boundaries dataset as well as this one, or just the law enforcement polygon boundaries? (this pull request is for the law enforcement facilities points) Is that to say that the sheriff's office points on this layer are derived from the county boundary polygons, and the police dept points from the Muni boundaries?

Reply:
ahh, good question. yes, you are correct, the data process I outlined above is solely for the polygon boundaries (not this layer). thanks for keeping me in line on that.

Reply:
Good to know! I'll jot this down and add it in once I get to the law enforcement boundaries dataset.